skins/contrib/Mozilla/global.css

```diff
@@ -337,7 +337,7 @@ hr {
 #bugzilla-body {
     background: none;
     border: none;
-    color: #404040;
+    color: #202020;
```

Comment:
I am not sure about this, as it would affect the whole BMO. I would like to defer this to @dylanwh